### PR TITLE
Tf012update

### PR DESCRIPTION
**Pull request by snovikov**

### CODE REVIEW

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+###
+### Lints all generic and json files in the whole git repository
+###
+
+name: linting
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```
Comment (titanlien) [inline .github/workflows/linting.yml:12]: Should we also triggered by any branch ?

Comment (snovikov) [reply .github/workflows/linting.yml:12]: I don't think we need it. We already have it for PRs.
Keep in mind - you pay for every second spent in GitHub actions.

Comment (cytopia) [reply .github/workflows/linting.yml:12]: > Keep in mind - you pay for every second spent in GitHub actions.

Not for public repositories ;-)

**Review by cytopia — APPROVED** [review]

File: .github/workflows/linting.yml (hunk 1 above)
Comment (snovikov) [reply .github/workflows/linting.yml:12]: good point :D

**Pull request merged**